src/jit/hwintrinsicxarch.cpp

```diff
@@ -561,6 +558,9 @@ bool Compiler::isFullyImplmentedISAClass(InstructionSet isa)
         case InstructionSet_SSE3:
         case InstructionSet_SSSE3:
         case InstructionSet_SSE41:
+        case InstructionSet_SSE42:
+        case InstructionSet_AVX:
+        case InstructionSet_AVX2:
```

Comment:
Move AVX/AVX2/SSE4.1 to fully-implemented ISA for GCStress/JITStress.

Reply:
This is also being handled by https://github.com/dotnet/coreclr/pull/17184, the same comment I made there applies here

Reply:
Yes, I will remove here after #17184 gets merged.